db/migrate/20170519181110_add_description_to_collections.rb

```diff
@@ -0,0 +1,6 @@
+class AddDescriptionToCollections < ActiveRecord::Migration
+  def change
+    add_column :collections, :description, :text, default: ""
```

Comment:
It's all about timing, if that table is locked for a while and people are trying to write to it then it'll impact on the site. I guess test the run time on a snapshot for metrics. 

You can always backfill and add the default in 1 migration / transaction too. I believe this will avoid avoid any locks.
```
add_column :collections, :description, :text
Collection.update_all(description: "")
change_column_default :collections :description ""
```